### DISCUSSION TITLE
Shut off Hero 11 after nightlapse or star trails

### DISCUSSION THREAD

**Discussion by dschaef51** [gopro/labs#154] [Q&A]
I'm trying to set up a script to do a nightlapse or star trails for several hours in a remote location then have the camera shut off before dawn. Having the camera turn on at the appropriate time and record for the correct length of time seems to work ok but the camera is still on after completion of task (the camera is also connected to external power). Is the a command to shut down the camera? I couldn't find it.

**Comment by dnewman-gpsw** [comment]
There is a shutdown command. It can be found here: https://gopro.github.io/labs/control/actions/

**!**_time_**O** - Off, shutdown the camera.  

or just **!O** will immediately power off.

Make sure your capture is stopped before calling this one.
